### DISCUSSION TITLE
Where are transcripts stored locally?

### DISCUSSION THREAD

**Discussion by johnghill** [lstrzepek/obsidian-yt-transcript#28] [Q&A] [answered]
Where does YTranscript store the text displayed in the right pane labelled "View YouTube Transcript" ? I have done a full-text search of my Obsidian library but cannot locate a local copy of the transcript. Thank you.

**Comment by lstrzepek** [comment, marked as answer]
Hi @johnghill 
It is not stored anywhere it is only in obsidian pane as a text if pane or obsidian is closed then all is gone.
